third_party/BUILD

```diff
@@ -18,10 +18,20 @@ load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 package(default_visibility = ["//visibility:public"])
 
 exports_files([
-    "empty.jar",
     "empty.jdeps",
 ])
 
+genrule(
+    name = "empty",
+    cmd = "cp $(location empty-definately-not-binary-jar.txt) $@",
+    srcs = [
+      "empty-definately-not-binary-jar.txt"
```

Comment:
😆

Reply:
I'm curious to see what the scanner thinks.